amplipi/streams.py

```diff
@@ -1321,6 +1321,216 @@ def info(self) -> models.SourceInfo:
     return source
 
 
+MUSIC_EXTENSIONS = ('.mp3', '.wav', '.aac', '.m4a', '.flac', '.aiff')
+
+
+class USBStick(BaseStream, Browsable):
```

Comment:
I would suggest making this a persistent stream so that an announcement can be played over it.